include/SSVOpenHexagon/Global/Config.hpp

```diff
@@ -19,6 +20,39 @@ class Trigger;
 
 namespace hg::Config
 {
+struct Version
+{
+    int major;
+    int minor;
+    int micro;
+
+    Version& operator=(const Version& o)
+    {
+        major = o.major;
+        minor = o.minor;
+        micro = o.micro;
+        return *this;
+    }
```

Comment:
```suggestion
```

This should be automatically generated by the compiler.

Reply:
Interesting. Did not know this.